CONTRIBUTING.md

```diff
@@ -94,14 +95,12 @@ If you are modifying JavaScript files, we also recommend:
 
 ## How-to
 
-### How to add a new JSON Schema
+### How to add a JSON Schema that's local to this repository
```

Comment:
The link on line 9 breaks due to this change I believe, since it no longer matches the header.

Reply:
thanks for the catch!